js/apple2.ts

```diff
@@ -32,27 +32,27 @@ import { Restorable, rom } from './types';
 import { processGamepad } from './ui/gamepad';
 
 export interface Apple2Options {
-    characterRom: string;
-    enhanced: boolean,
-    e: boolean,
-    gl: boolean,
-    rom: string,
-    canvas: HTMLCanvasElement,
-    tick: () => void,
+    characterRom: string
+    enhanced: boolean
+    e: boolean
+    gl: boolean
+    rom: string
+    canvas: HTMLCanvasElement
+    tick: () => void
```

Comment:
Looks like this is removing the member separators whereas below you add them. Same below.